### PR TITLE
3.2 changes

### PR DESCRIPTION
**Pull request by tapir**

### CODE REVIEW

File: v3.2/glfw/c_glfw_windows.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package glfw
+
+/*
+// Bug fix for 6l/8l on Windows, see:
```
Comment (tapir) [inline v3.2/glfw/c_glfw_windows.go]: Do we still need to include these?

**Comment by tapir** [comment]
https://github.com/glfw/glfw/issues/783
So for now I disabled EGL for windows and X11

**Comment by tapir** [comment]
At this point I think we can merge this. The remaining Vulkan methods can be added later. I'm not even sure if they are needed for most.

**Comment by tapir** [comment]
If there are no complaints I will merge this in 2 days. It's been 20 days since 3.2 release so I guess it's time.

**Comment by dmitshur** [comment]
@tapir Sounds good. Even if there are issues in the 3.2 package, those can be reported and fixed after it's merged.

Still, I'll try to review this PR in 2 days from now, can you wait until Saturday?

**Comment by tapir** [comment]
Sure no problem :+1:

**Comment by dmitshur** [comment]
Installing `v3.2/glfw` has a few warnings on OS X.

```
glfw $ go install -v ./v3.1/glfw
github.com/go-gl/glfw/v3.1/glfw

glfw $ go install -v ./v3.2/glfw
github.com/go-gl/glfw/v3.2/glfw
# github.com/go-gl/glfw/v3.2/glfw
v3.2/glfw/native_darwin.go:12:37: warning: incompatible pointer types passing 'GLFWwindow *' (aka 'struct GLFWwindow *') to parameter of type 'GLFWmonitor *' (aka 'struct GLFWmonitor *') [-Wincompatible-pointer-types]
v3.2/glfw/glfw/include/GLFW/glfw3native.h:198:60: note: passing argument to parameter 'monitor' here
v3.2/glfw/native_darwin.go:12:9: warning: cast to 'void *' from smaller integer type 'CGDirectDisplayID' (aka 'unsigned int') [-Wint-to-void-pointer-cast]
```

I'll look into what it'd take to fix them later, just sharing findings for now.

**Comment by dmitshur** [comment]
I tested it with all my projects and everything worked well. I'll review the code up next.

**Comment by dmitshur** [comment]
Okay, onto fixing those warnings. First, I found a copy-paste error:

```
void *workaround_glfwGetCocoaMonitor(GLFWwindow *w) {
```

`glfwGetCocoaMonitor` should accept a parameter of type `GLFWmonitor *`, not `GLFWwindow *`.

That fixes first warning.

However, `glfwGetCocoaMonitor` returns type `CGDirectDisplayID`, not `id` like the other two:

http://www.glfw.org/docs/latest/group__native.html#gaf22f429aec4b1aab316142d66d9be3e6

So the workaround isn't needed at all. We can just call `C.glfwGetCocoaMonitor` directly.

I'll push a fix to this branch.

**Comment by dmitshur** [comment]
I reviewed the code and did not spot any other issues. I think this is good to merge now. We can start using the new version and should anything come up, it can be fixed up in future PRs. Same goes for any missing new features/APIs.

LGTM.

I think you should consider squashing and rebasing it before merging, but it's okay to merge as is if you want to preserve the development history of 3.2.

**Comment by tapir** [comment]
Done. Thank you @shurcooL 
I'm merging this now.

**Pull request merged**